### PR TITLE
Movement GUI [4/4] Calibration Wizard

### PR DESCRIPTION
**Pull request by maltewae**
Adds a wizard to calibrate stages, i.e. define the axis rotation and define the coordinate pairings.

No new logic was added, only existing logic was simplified and moved.

GUI tests will follow.

### CODE REVIEW

File: LabExT/View/Movement/MovementWizard.py [hunk 1]
```diff
@@ -526,3 +565,381 @@ def _build_entry_with_label(
...
+        if not self._confirm_wiggle(chip_axis):
+            return
+
+        try:
+            run_with_wait_window(
+                self.wizard,
+                f"Wiggle {chip_axis} of {calibration}",
+                lambda: calibration.wiggle_axis(chip_axis))
+        except RuntimeError as e:
+            messagebox.showerror(
```
Comment (marcoep) [inline LabExT/View/Movement/MovementWizard.py:709]: add logger message here please

File: LabExT/View/Movement/MovementWizard.py [hunk 2]
```diff
@@ -526,3 +565,381 @@ def _build_entry_with_label(
...
+        ----------
+        frame : CustomFrame
+            Instance of a customized Tkinter frame.
+        """
+        frame.title = "Calibrate Stage to enable absolute movement"
+
+        Label(
+            frame,
+            text="To move the stages absolutely in chip coordinates, define at least 3 stage-chip-coordinate pairings to calculate the rotation. \n" +
+            "Note: After one coordinate pairing TODO. Therefore this is only an approximation.").pack(
```
Comment (marcoep) [inline LabExT/View/Movement/MovementWizard.py:775]: what does this comment mean?

**Review by marcoep — CHANGES_REQUESTED** [review]

File: LabExT/View/Movement/MovementWizard.py (hunk 2 above)
Comment (maltewae) [reply LabExT/View/Movement/MovementWizard.py:775]: Oops! That was a hint for me to add a description. I will change it. 😄

File: LabExT/View/Movement/MovementWizard.py (hunk 1 above)
Comment (maltewae) [reply LabExT/View/Movement/MovementWizard.py:726]: Added 👍🏼

File: LabExT/View/Movement/MovementWizard.py (hunk 2 above)
Comment (maltewae) [reply LabExT/View/Movement/MovementWizard.py]: Fixed!

**Pull request merged**